### PR TITLE
fixes plugin not starting/working

### PR DESCRIPTION
**Pull request by accelyte**

### CODE REVIEW

File: HideSidebar.plugin.js [hunk 1]
```diff
@@ -8,21 +8,24 @@
...
 
-    const sidebarSelector = createSelector(BdApi.findModuleByProps("hasNotice").sidebar)
-    const guildBarSelector = createSelector(BdApi.findModuleByProps("hasNotice").guilds)
-    const sidebarBtn = document.createElement('span');
+    const sidebarSelector = createSelector(
+        BdApi.findModuleByProps("guilds").sidebar
+    );
+    const guildBarSelector = createSelector(
+        BdApi.findModuleByProps("guilds").guilds
+    );
```
Comment (accelyte) [inline HideSidebar.plugin.js:21]: main changes are here, the rest are auto-formatting changes (prettier didn't like what it saw).

**Comment by adam-omar20** [comment]
> 

I don't understand all these complications. Can you upload the file after the final modification?

**Comment by accelyte** [comment]
You can get the raw file contents [here](https://raw.githubusercontent.com/Pieloaf/BetterDiscordPlugins/b9d2f415106e599839ff0d456d69d62c9d9b7fdd/HideSidebar.plugin.js).

**Comment by Pieloaf** [comment]
`BdApi.findModuleByProps("guilds").sidebar` is returning undefined. Taking a look at the API docs now to see if I can get the sidebar another way

**Comment by Pieloaf** [comment]
`BdApi.findModuleByProps("guilds").guilds` was returning a class name but it couldn't be found in the DOM (Qb also advised i use a similar selector when I first started making this but had the same issue with some class names not found)

but I updated the selectors to the following, and it works again:
```    const sidebarSelector = createSelector(
        BdApi.findAllModules(m => m.sidebar)[1].sidebar
    );
    const guildBarSelector = createSelector(
        BdApi.findAllModules(m => m.guilds)[1].guilds
    );

**Comment by accelyte** [comment]
That's interesting, maybe it has to do with my setup then, since it does work on my device.

**Comment by Pieloaf** [comment]
Yeah, idk what's up with it, like it returns a class name but trying to select it returns undefined :/

**Comment by accelyte** [comment]
I can't say for sure, but maybe it's an artefact left behind during the Discord build process. To be honest I came across my fix by chance. This is the first time I've worked with  BD as well so...🤷‍♂️

**Pull request closed without merging**